ts-obs/tests/installation_test.go

```diff
@@ -0,0 +1,145 @@
+package tests
+
+import (
+    "fmt"
+    "os/exec"
+    "testing"
+    "time"
+)
+
+func testInstall(t testing.TB, name string, filename string) {
+    var install *exec.Cmd
+
+    if name == ""  && filename == "" {
+        fmt.Println("Running 'ts-obs install'")
```

Comment:
all these prints will eventually have to be deleted I think.

Reply:
I like them :\

Reply:
I think the quit flag is a good idea.

Reply:
Using the default go test -v flag instead.